### PR TITLE
[SP-4298] Backport of PRD-5569 - Published Report's Filename is Displ…

### PR DESCRIPTION
**Pull request by ricardosilva88**
…ayed instead of Title in PUC's Browse folders (7.1 Suite)

@cravobranco 
@pentaho-lmartins

### CODE REVIEW

File: designer/report-designer-extension-pentaho/src/main/java/org/pentaho/reporting/designer/extensions/pentaho/repository/util/PublishUtil.java [hunk 1]
```diff
@@ -169,6 +168,17 @@ public static int publish( final byte[] data, final String path, final Authentic
     return responseCode;
   }
 
+  /**
+   * We need to keep the options of report. please use {@link #publish(byte[], String, AuthenticationData, Properties)}
+   * We keep the method for backward compatibility
+   *
+   * @since pentaho 8.1
```
Comment (pentaho-lmartins) [inline designer/report-designer-extension-pentaho/src/main/java/org/pentaho/reporting/designer/extensions/pentaho/repository/util/PublishUtil.java]: Remove this comment in the backport.

File: libraries/libpensol/src/main/java/org/pentaho/reporting/libraries/pensol/PublishRestUtil.java [hunk 2]
```diff
@@ -147,4 +152,30 @@ public int publishFile( String repositoryPath, String fileName, InputStream file
     }
     return responseCode;
   }
+
+  /**
+   * We need to keep the options of report. please use {@link #publishFile(String, byte[], Properties)}
+   * We keep the method for backward compatibility
+   *
+   * @since pentaho 8.1
```
Comment (pentaho-lmartins) [inline libraries/libpensol/src/main/java/org/pentaho/reporting/libraries/pensol/PublishRestUtil.java]: Remove this comment in the backport.

File: libraries/libpensol/src/main/java/org/pentaho/reporting/libraries/pensol/PublishRestUtil.java [hunk 3]
```diff
@@ -147,4 +152,30 @@ public int publishFile( String repositoryPath, String fileName, InputStream file
...
+    Properties fileProperties =  new Properties();
+    fileProperties.setProperty( OVERWRITE_FILE_KEY, String.valueOf( overwriteIfExists ) );
+    return publishFile( filePath, data, fileProperties );
+  }
+
+  /**
+   * We need to keep the options of report. please use {@link #publishFile(String, String, InputStream, Properties)}
+   * We keep the method for backward compatibility
+   *
+   * @since pentaho 8.1
```
Comment (pentaho-lmartins) [inline libraries/libpensol/src/main/java/org/pentaho/reporting/libraries/pensol/PublishRestUtil.java]: Remove this comment in the backport.

**Comment by ricardosilva88** [comment]
@pentaho-lmartins PR updated

**Pull request merged**